catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/SparkServlet.java

```diff
@@ -129,6 +139,21 @@ public void init(ServletConfig config) throws ServletException {
                 false);
     }
 
+    private void registerProximityFactory() {
```

Comment:
I'm not sure that that is necessary.  This function is only called during the `init()` of the Catalog UI; the FunctionFactory should be registered with GeoTools on initialization